reinforcement_learning/rl_cartpole_batch_coach/rl_cartpole_batch_coach.ipynb

```diff
@@ -0,0 +1,712 @@
+{
```

Comment:
<p>It would be good to add a sentence prior to this cell, calling out this is the baseline.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>revised</p>